doc/introduction/pulse.rst

```diff
@@ -0,0 +1,339 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_pulse:
+
+Pulse control
+=============
+
+Pulse control is used in a variety of quantum systems for low-level control of quantum operations. A
+time-dependent electromagnetic field tuned to the characteristic energies is applied,
+leading to a time-dependent Hamiltonian interaction :math:`H(t)`. We call driving the system with such an
+electromagnetic field for a fixed time window a *pulse sequence*. These pulse sequences can then be tuned to
+implement the higher level gates used for quantum computation.
+
+The :mod:`~.pulse` module provides functions and classes used to simulate pulse-level control of quantum
+systems. It contains a :class:`~.ParametrizedHamiltonian` class, which can be used to define the time-dependent
+Hamiltonian describing the interaction between the applied pulses and the system. A
+:class:`~.ParametrizedHamiltonian` instance can be used to create a :class:`~.ParametrizedEvolution`
+(an :class:`~.Operation`) that provides the time evolution under the Hamiltonian. The :mod:`~.pulse` module also
+includes several convenience functions for defining pulse envelopes.
+
+The :mod:`~.pulse` module relies on the external package `JAX <https://jax.readthedocs.io/en/latest/>`_, which
+requires separate installation. The module is written for ``jax`` and will not work with other machine learning
+frameworks typically encountered in PennyLane.
+
+
+
+Creating a parametrized Hamiltonian
+-----------------------------------
+
+The :mod:`~.pulse` module provides a framework to create a time-dependent Hamiltonian of the form
+
+.. math:: H(p, t) = H_\text{drift} + \sum_j f_j(p, t) H_j
+
+with constant operators :math:`H_j` and scalar functions :math:`f_j(p, t)` that may depend on
+parameters :math:`p` and time :math:`t`.
+
+Defining a :class:`~.ParametrizedHamiltonian` requires coefficients and operators. In the example below, we define a
+Hamiltonian with a single drift term, and two parametrized terms.
```

Comment:
Super nit-picky: it sounds like the following code cell would create the Hamiltonian, but then only the cell after that actually creates it. Really minor, though.